base/asyncevent.jl

```diff
@@ -206,6 +208,14 @@ end
 
 isopen(t::Union{Timer, AsyncCondition}) = @atomic :acquire t.isopen
 
+"""
+    close(t::Union{Timer, AsyncCondition})
+
+Close an object `t`. Once a timer or condition is closed, it will not produce
```

Comment:
Continuing from above, how about
```suggestion
Close an object `t` and thus mark it as inactive. Once a timer or condition is inactive, it will not produce
```